core/domain/user_services.py

```diff
@@ -388,3 +388,99 @@ def get_email_preferences(user_id):
             if email_preferences_model is None
             else email_preferences_model.site_updates)
     }
+
+
+class UserContributions(object):
+    """Value object representing a user's contributions."""
+    def __init__(
+            self, user_id, created_explorations, edited_explorations):
+        self.user_id = user_id
+        self.created_explorations = created_explorations
```

Comment:
Sorry -- I just realized that the name for this is wrong. You're actually storing a list of exploration_ids, so these should really be created_exploration_ids and edited_exploration_ids. Otherwise there is a potential confusion about whether the elements represent exploration ids or exploration objects.

Again, apologies for catching this so late, but I think it does need to be changed -- it's much harder to update a model definition after it's been launched.